src/utils/content-charset.js

```diff
@@ -0,0 +1,155 @@
+const CHARSET_RE      = /(?:^|;)\s*charset=(.+)(?:;|$)/i;
+const META_CHARSET_RE = /charset ?= ?['"]?([^ ;"']*)['"]?/i;
+// NOTE: HTTP 1.1 specifies ISO-8859-1 as a default charset
+// (see: http://www.w3.org/International/O-HTTP-charset.en.php).
+const DEFAULT_CHARSET         = 'iso-8859-1';
+const CHARSET_BOM_LIST        = [
```

Comment:
Use new lines to separate object and array properties to achieve better formatting.